soc/nordic/nrf54h/soc.c

```diff
@@ -126,22 +126,15 @@ bool z_arm_on_enter_cpu_idle(void)
 }
 #endif
 
-static int nordicsemi_nrf54h_init(void)
+void soc_early_init_hook(void)
 {
-	int err;
-
 	sys_cache_instr_enable();
 	sys_cache_data_enable();
 
 	power_domain_init();
 
 	trim_hsfll();
 
-	err = dmm_init();
```

Comment:
After this is removed, also the following line is no longer needed: https://github.com/zephyrproject-rtos/zephyr/blob/12f4ac9dbfbce7ea0fdef2dcd973116ebb78dc13/soc/nordic/nrf54h/soc.c#L24

Reply:
fixed in https://github.com/zephyrproject-rtos/zephyr/pull/89333